packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/utils/check_index_algorithm.ts

```diff
@@ -12,11 +12,17 @@ import type { IndexMapping } from '@kbn/core-saved-objects-base-server-internal'
  * The list of values returned by `checkIndexCurrentAlgorithm`.
  *
  * - `zdt`
+ * - `zdt-partially-migrated`
  * - `v2-compatible`
  * - `v2-incompatible`
  * - `unknown`
```

Comment:
WYDT about treating these entries as "headings" and providing a short explanation where appropriate. Something like:

```
- `v2-partially-migrated`
   This can happen when we detected a `v2-compatible` index, updated index metadata and got interrupted at a later stage. In this case we retry the `v2-compatible` migration.
```